samples/pubsub/message-driven/Core_6/Publisher/Program.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+static class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.PubSub.MessageDrivenPublisher";
+        var endpointConfiguration = new EndpointConfiguration("Samples.PubSub.MessageDrivenPublisher");
+        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+        endpointConfiguration.UseTransport<MsmqTransport>();
+
+        endpointConfiguration.SendFailedMessagesTo("error");
+        endpointConfiguration.EnableInstallers();
+
+        var endpointInstance = await Endpoint.Start(endpointConfiguration)
+            .ConfigureAwait(false);
+        await Start(endpointInstance)
```

Comment:
The name start feels confusing that there is a start call on the line above. How about `PublishOrderEvent` or something?

Reply:
fair point, took that from the existing sample but sounds like a good improvement 👍